calibrations/facilities/photometric_standards_facility.py

```diff
@@ -202,6 +200,9 @@ def _build_instrument_config(self):
                 "offset_dec": 0,
                 "defocus": 0
             }
+            base_narrowband_position = None
+            if 'narrobands' in self.cleaned_data and self.cleaned_data['narrowbands']:
```

Comment:
is this ('narrobands' vs. 'narrowbands') intentional?

Reply:
yeah it was "diffusers" before, so I kept it plural. It seemed to be a setting to set all diffusers/narrowbands to that value rather then setting them each individually.

Reply:
but it is missing the 'w'

Reply:
touche, that's why we do code reviews :)